specification/securityinsights/resource-manager/Microsoft.SecurityInsights/stable/2024-03-01/DataConnectors.json

```diff
@@ -651,6 +661,118 @@
       },
       "type": "object"
     },
+    "TiTaxiiDataConnector": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataConnector"
+        }
+      ],
+      "description": "Data connector to pull Threat intelligence data from TAXII 2.0/2.1 server",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/TiTaxiiDataConnectorProperties",
+          "description": "Threat intelligence TAXII data connector properties.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "type": "object",
+      "x-ms-discriminator-value": "ThreatIntelligenceTaxii"
+    },
+    "TiTaxiiDataConnectorDataTypes": {
+      "description": "The available data types for Threat Intelligence TAXII data connector.",
+      "properties": {
+        "taxiiClient": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/DataConnectorDataTypeCommon"
+            }
+          ],
+          "description": "Data type for TAXII connector.",
+          "type": "object"
+        }
+      },
+      "type": "object",
+      "required": [
+        "taxiiClient"
+      ]
+    },
+    "TiTaxiiDataConnectorProperties": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/DataConnectorTenantId"
+        }
+      ],
+      "description": "Threat Intelligence TAXII data connector properties.",
+      "properties": {
+        "workspaceId": {
+          "description": "The workspace id.",
+          "type": "string"
+        },
+        "friendlyName": {
+          "description": "The friendly name for the TAXII server.",
+          "type": "string"
+        },
+        "taxiiServer": {
+          "description": "The API root for the TAXII server.",
+          "type": "string"
+        },
+        "collectionId": {
+          "description": "The collection id of the TAXII server.",
+          "type": "string"
+        },
+        "userName": {
+          "description": "The userName for the TAXII server.",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password for the TAXII server.",
+          "type": "string"
+        },
+        "taxiiLookbackPeriod": {
+          "description": "The lookback period for the TAXII server.",
```

Comment:
>period [](http://example.com/codeflow?start=39&length=6)

naming nitpick, I'd call it an Interval rather than a Period, since its not really periodic in the context of doing one look back, is it?